roles/tripleo_cluster_set_container_cli/tasks/set_container_cli.yaml

```diff
@@ -0,0 +1,22 @@
+- name: Get ceph fsid
+  command: "awk '/fsid/ {print $3}' /etc/ceph/ceph.conf"
+  register: fsid
+  run_once: true
+  changed_when: false
+
+- name: Set fsid fact
+  set_fact:
+    fsid: "{{ fsid.stdout }}"
+
+- name: Set client fact
+  set_fact:
+    ceph_cli: "{{ container_cli }} run --rm -it {{ container_options }} -v {{ ceph_config_home}}:/etc/ceph:z
+               {{ ceph_container_ns }}/{{ ceph_container_image }}:{{ ceph_container_tag }} ceph --fsid {{ fsid }}
+               -c {{ ceph_dot_conf }} -k {{ ceph_keyring }}"
+  run_once: true
```

Comment:
FWIW I'm getting no OSDs in my env but I don't think it's related to this change as both the old and new ceph_cli definitions have the same result.

```
[root@controller-0 ~]# podman run --rm -it --net=host --ipc=host -v /etc/ceph:/etc/ceph:z docker.io/ceph
/ceph:v15 ceph --fsid 1b2cb492-a229-11ea-bef0-244200624c1c -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.clie
nt.admin.keyring orch apply osd --all-available-devices                                                
No pending deployments.
[root@controller-0 ~]#

[root@controller-0 ~]# cephadm shell --fsid 1b2cb492-a229-11ea-bef0-244200624c1c -c /etc/ceph/ceph.conf 
-k /etc/ceph/ceph.client.admin.keyring -- ceph orch apply osd --all-available-devices                   
INFO:cephadm:Using recent ceph image docker.io/ceph/ceph:v15
No pending deployments.
[root@controller-0 ~]# 
```

```
[root@controller-0 ~]# podman run --rm -it --net=host --ipc=host -v /etc/ceph:/etc/ceph:z docker.io/ceph
/ceph:v15 ceph --fsid 1b2cb492-a229-11ea-bef0-244200624c1c -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.clie
nt.admin.keyring orch daemon add osd compute-0:/dev/sda,/dev/sdb,/dev/sdc,/dev/sdd
Created no osd(s) on host compute-0; already created?
[root@controller-0 ~]# 

[root@controller-0 ~]# cephadm shell --fsid 1b2cb492-a229-11ea-bef0-244200624c1c -c /etc/ceph/ceph.conf 
-k /etc/ceph/ceph.client.admin.keyring -- ceph orch daemon add osd compute-0:/dev/sda,/dev/sdb,/dev/sdc,
/dev/sdd                                                                                                
INFO:cephadm:Using recent ceph image docker.io/ceph/ceph:v15
Created no osd(s) on host compute-0; already created?
[root@controller-0 ~]# 
```